devenv/dev-dashboards/datasource-loki/loki_query_splitting.json

```diff
@@ -0,0 +1,900 @@
+              "align": "auto",
+              "cellOptions": {
+                "type": "auto"
+              },
+              "inspect": false
+            },
+            "mappings": [],
+            "thresholds": {
+              "mode": "absolute",
+              "steps": [
+                {
+                  "color": "green",
+                  "value": null
+                },
+                {
+                  "color": "red",
+                  "value": 80
+                }
+              ]
+            }
+          },
+          "overrides": []
+        },
+        "gridPos": {
+          "h": 8,
+          "w": 12,
+          "x": 12,
+          "y": 28
+        },
+        "id": 10,
+        "options": {
+          "cellHeight": "sm",
+          "footer": {
+            "countRows": false,
+            "fields": "",
+            "reducer": [
+              "sum"
+            ],
+            "show": false
+          },
+          "showHeader": true
+        },
+        "pluginVersion": "10.2.0-61469",
+        "targets": [
+          {
+            "datasource": {
+              "type": "loki",
+              "uid": "gdev-loki"
+            },
+            "editorMode": "code",
+            "expr": "{place=\"moon\"}",
+            "maxLines": 5,
+            "queryType": "range",
+            "refId": "do-not-chunk"
+          }
+        ],
+        "title": "Logs with extract key=value and organize, no splitting",
+        "transformations": [
+          {
+            "id": "extractFields",
+            "options": {
+              "format": "auto",
+              "keepTime": true,
+              "replace": true,
+              "source": "Line"
+            }
+          },
+          {
+            "id": "organize",
+            "options": {
+              "excludeByName": {
+                "31": true,
+                "32": true,
+                "33": true,
+                "34": true,
+                "35": true,
+                "36": true,
+                "37": true,
+                "38": true,
+                "39": true,
+                "40": true,
+                "41": true,
+                "42": true,
+                "43": true,
+                "44": true,
+                "45": true,
+                "46": true,
+                "2023-09-20T14": true,
+                "33+00": true,
+                "caller": true,
+                "main.go": true,
+                "t": true,
+                "ts": true
+              },
+              "indexByName": {},
+              "renameByName": {
+                "level": "nivel",
+                "ts": ""
+              }
+            }
+          }
+        ],
+        "type": "table"
+      }
+    ],
+    "refresh": "",
+    "schemaVersion": 38,
+    "tags": [],
+    "templating": {
+      "list": []
+    },
+    "time": {
+      "from": "now-6h",
+      "to": "now"
+    },
+    "timepicker": {},
+    "timezone": "",
+    "title": "Loki Query splitting",
```

Comment:
```suggestion
    "title": "Datasource tests - Loki query splitting",
```

Based on convention for data sources
<img width="272" alt="image" src="https://github.com/grafana/grafana/assets/30407135/d0b2ffb8-e7a0-48f3-a01c-d26b1218bb69">